### DISCUSSION TITLE
help please? how to change from AS2 to AS4??

### DISCUSSION THREAD

**Discussion by ervin09** [OxalisCommunity/oxalis#698] [General]
hi guys 

i've deployed oxalis using the latest docker image, but i don't see any thing how to change from AS2 to AS4....
i tried to copy the lastest file oxalis-as4.jar inside the container ..but it only leads to the container to be break... can someone point me in the right direction please.... i've been spining for hours now..

version.oxalis: 6.7.1-SNAPSHOT
version.java: 11.0.16
mode: TEST
build.id: dcd7f26c692422857e3a826820a6702c1e2980c3
build.tstamp: 20.10.2024 @ 16:32:07 UTC

**Comment by seciq** [comment]
Have you tried to run this container: https://hub.docker.com/r/norstella/oxalis-as4
It has the AS4 functionality built in.
